core/src/main/scala/akka/persistence/cassandra/journal/TagWriter.scala

```diff
@@ -55,22 +51,34 @@ import scala.util.{ Failure, Success, Try }
     pidTagSequenceNr: TagPidSequenceNr
   )
 
-  val timeUuidOrdering = new Ordering[UUID] {
-    override def compare(x: UUID, y: UUID) = UUIDComparator.comparator.compare(x, y)
-  }
+  private[akka] case class TagWrite(tag: Tag, serialised: Vector[Serialized]) extends NoSerializationVerificationNeeded
 
-  private case object InternalFlush
-  private case object FlushKey
+  /*
+   * Reset pid tag sequence numbers to the given [[TagProgress]] and discard any messages for the given persistenceId
+   */
+  private[akka] case class ResetPersistenceId(persistenceId: PersistenceId, tag: Tag, progress: TagProgress)
```

Comment:
Hmm, why do we need the separate pid when it is already in the `TagProgress`?

Reply:
We don't, i'll remove it